.codeclimate.yml

```diff
@@ -81,5 +81,6 @@ exclude_patterns:
   - "**/pytest_plugin/"
   - "**/sandbox/"
   - packages/models-library/src/models_library/utils/_original_fastapi_encoders.py
+  - services/resource-usage-tracker/src/simcore_service_resource_usage_tracker/modules/db/repositories/resource_tracker.py
```

Comment:
Hey guys, I kindly disagree with you in this situation. The reason why the codeclimate was complaining is that I went over 500 lines of code. This is a repository! therefore it is all just about SQL select statements from the database which of course can be long, but is really clear and it does not add any complexity. I think this tool is great and we should use it (but do not listen to it blindly, because in this use-case for me, it would make no sense to artificially divide this file into more, I would argue that would increase the complexity)